app/assets/javascripts/app/form-field-format.js

```diff
@@ -0,0 +1,21 @@
+import Formatter from 'formatter.js-pebble';
+
+
+function formatForm() {
+  const formats = [
+    ['input[type="tel"]', '+1 ({{999}}) {{999}}-{{9999}}'],
```

Comment:
I know we only supporting US numbers right now.. but in the future we might support others.  I suppose code changes will be needed at that point anyway, and this doesn't lock us in in some irreversible way...  so lgtm